src/reactnative/builds/algoliasearch.reactnative.js

```diff
@@ -24,6 +24,12 @@ function algoliasearch(applicationID, apiKey, opts) {
 
   opts._ua = opts._ua || algoliasearch.ua;
 
+  opts.timeouts = opts.timeouts || {
+    connect: 2 * 1000,
+    read: 4 * 1000,
```

Comment:
It's 5 minutes on every other API clients. 
But maybe it'll make more sense to reduce it for the front-end implementations?